src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -7,29 +7,70 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-A sidechain is a separate blockchain which runs in parallel to Ethereum Mainnet and operates independently. It has its own [consensus algorithm](/developers/docs/consensus-mechanisms/) (e.g. [proof-of-authority](https://wikipedia.org/wiki/Proof_of_authority), [Delegated proof-of-stake](https://en.bitcoinwiki.org/wiki/DPoS), [Byzantine fault tolerance](https://decrypt.co/resources/byzantine-fault-tolerance-what-is-it-explained)). It is connected to Mainnet by a two-way bridge.
+A sidechain is a separate blockchain which runs in parallel to Ethereum; it is connected to Mainnet by a two-way bridge. Sidechains have separate [consensus algorithms](/developers/docs/consensus-mechanisms/), which are often designed for faster processing times. Using a sidechain involves trade-offs, though, as they do not inherit Ethereum's security properties, unlike [layer 2 scaling solutions](/layer-2/). 
 
-What makes a sidechain particularly exciting is that the chain works the same as the main Ethereum chain because it's based on [the EVM](/developers/docs/evm/). It doesn't use Ethereum, it is Ethereum. This means if you want to use your [dapp](/developers/docs/dapps/) on a sidechain, it's just a matter of deploying your code to this sidechain. It looks, feels, and acts just like Mainnet – you write contracts in Solidity, and interact with the chain via the Web3 API.
+## How do sidechains work? {#how-do-sidechains-work}
 
-## Prerequisites {#prerequisites}
+### Consensus algorithms {#consensus-algorithms}
 
-You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/).
+As mentioned, sidechains use alternative consensus algorithms that are different from Ethereum's proof-of-work (PoW) or proof-of-stake (PoS). Some examples of consensus algorithms used on sidechains include:
 
-## Pros and cons {#pros-and-cons}
+- [Proof-of-authority](https://wikipedia.org/wiki/Proof_of_authority)
+- [Delegated proof-of-stake](https://en.bitcoinwiki.org/wiki/DPoS)
+- [Byzantine fault tolerance](https://decrypt.co/resources/byzantine-fault-tolerance-what-is-it-explained). 
+
+Sidechain nodes (called validators) are responsible for ordering and processing transactions, storing the blockchain state, and securing the network. In some cases, validators may be responsible for approving users' entries and exits between the sidechain and Ethereum Mainnet. 
+
+### EVM compatibility {#evm-compatibility}
+
+What makes a sidechain particularly exciting is that the chain works the same as the main Ethereum chain because it's based on [the EVM](/developers/docs/evm/). It doesn't merely use Ethereum, it _is_ Ethereum. 
```

Comment:
I'm a bit cautious of saying an emphasizing "it _is_ Ethereum." Feel like this could lead to thinking that because a chain is EVM compatible that it "is Ethereum" and adding to the confusion that it's all the same security assurances when it isn't. 

Perhaps we focus on how the language used to interpret smart contracts is the same, allowing for compatibility.